packages/jest-reporters/src/Status.ts

```diff
@@ -63,16 +64,16 @@ class CurrentTestList {
  * from the terminal.
  */
 export default class Status {
-  _cache: ?{content: string, clear: string};
-  _callback: () => void;
+  _cache: {content: string; clear: string} | null;
```

Comment:
ts tells me that `_height` and `_lastUpdated` are unused. In fact both are set once and never used again. Should i just remove them?

Reply:
yeah!